experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -77,6 +77,24 @@ class Flowables {
         std::forward<OnSubscribe>(function)));
   }
 
+  template <typename T>
+  static Reference<Flowable<T>> empty() {
+    auto lambda = [](Subscriber<T>& subscriber, int64_t) {
+      subscriber.onComplete();
+      return std::make_tuple(static_cast<int64_t>(0), true);
```

Comment:
I think `int64_t{0}` is better than `static_cast<int64_t>(0)`.  `int64_t{n}` has the benefit of being a compiler error if `n` cannot be guaranteed to fit within the bounds of an `int64_t`.